index.html

```diff
@@ -289,6 +289,15 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <section class="contact">
+        <h3 id="form-header"> I'm always interested in hearing about new projects, so if you'd like to chat please get in</h3>
+        <form class="form" action=" https://formspree.io/f/xyylppbj" method="POST">
+          <input type="text" name="Names"  id="name" maxlength='30' placeholder="Name">
+          <input type="email" name="E-mail" id="email" placeholder="Email">
+          <textarea name="Message" rows="5" id="message" placeholder="Your message"></textarea>
+          <input type="submit" value="Get In Touch">
```

Comment:
- [ ] As per project requirement there should `button` interaction. when user hover, active, disable, enable. Let's try to fix this. 

![image](https://user-images.githubusercontent.com/53019916/124177686-2e4f5000-dace-11eb-80ac-f46528d52d56.png)

Reply:
Please add the required change the the previous TSE mentioned about the button interaction.